src/OrchardCore/OrchardCore.Abstractions/OrchardCoreConstants.cs

```diff
@@ -28,4 +28,44 @@ public class Roles
 
         public const string Anonymous = "Anonymous";
     }
+
+    public class ConfigureOrder
+    {
+        public const int Default = 0;
+
+        // The configuration for authentication should be set up early, prior to any non-security modules.
+        public const int Authentication = -150;
+
+        // The reverse proxy should always be configured before the 'Authentication' and security initialization logic.
+        public const int ReverseProxy = Authentication * 2;
+
+        public const int Cors = ReverseProxy;
+
+        public const int Media = Default;
+
+        // Image cache overrides Media configurations and services.
+        // The order number should always be greater than Media module. 
+        public const int ImageSharpCache = Media + 5;
+
+        // Image cache overrides Media configurations and services.
+        // The order number should always be greater than Media module. 
+        public const int AzureImageSharpCache = Media + 5;
+
+        // Azure media storage overrides Media configurations and services.
+        // The order number should always be greater than Media module.
+        public const int AzureMediaStorage = Media + 10;
+
+        public const int DataProtection = Default;
+
+        // Azure DataProtection will override default data-protection configurations.
+        // The order number should always be greater than data protection modules. 
+        public const int AzureDataProtection = DataProtection + 10;
+
+        public const int HomeRoute = -150;
```

Comment:
Self-note: the home route probably should be greater than the Authentication. Did not change it here to avoid possible unknown issues. so maybe -10 is sufficient. Not really sure why this module even need to define an order and it does not override services. Probably `Default` 0 is even better